### PR TITLE
Feature/update schema version

### PR DESCRIPTION
**Pull request by blabadi**
**Description of changes**

**Type of Change**

- [ ] Bug
- [ ] Refactor
- [x] New Feature

**Checklist before requesting review:**

- [x] Check branch (code change PRs go to `develop` not master)
- [x] Manual testing
- [x] Regression tests completed and passing (double check number of tests).
- [x] Spelling has been checked.
- [x] Updated swagger docs accordingly (check it's still valid)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -97,6 +97,7 @@
     "express-validator": "^6.0.1",
     "jsonwebtoken": "^8.5.1",
     "lodash": "^4.17.14",
+    "mock-http-server": "^1.4.1",
```
Comment (blabadi) [inline package.json]: to mock lectern

File: src/lectern-client/schema-rest-client.ts [hunk 2]
```diff
@@ -19,9 +19,19 @@ export const schemaClient: SchemaServiceRestClient = {
       L.debug(`in fetch schema ${version}`);
       const result = delay(1000);
       const dictionary = await result(() => {
-        const dictionaryJson = require(schemaSvcUrl.substring(7, schemaSvcUrl.length));
-        return dictionaryJson;
+        const dictionaries: SchemasDictionary[] = require(schemaSvcUrl.substring(
+          7,
+          schemaSvcUrl.length,
+        )) as SchemasDictionary[];
+        const dic = dictionaries.find((d: any) => d.version == version);
```
Comment (jaserud) [inline src/lectern-client/schema-rest-client.ts]: and I guess we assume that all the schema in stub schema array have same `name`?? (might help to note that too)

File: src/lectern-client/schema-rest-client.ts [hunk 3]
```diff
@@ -19,9 +19,19 @@ export const schemaClient: SchemaServiceRestClient = {
       L.debug(`in fetch schema ${version}`);
       const result = delay(1000);
       const dictionary = await result(() => {
-        const dictionaryJson = require(schemaSvcUrl.substring(7, schemaSvcUrl.length));
-        return dictionaryJson;
+        const dictionaries: SchemasDictionary[] = require(schemaSvcUrl.substring(
```
[the remark this answers is not in the document]
Comment (blabadi) [reply src/lectern-client/schema-rest-client.ts]: I've updated all the stub schema files in the code

File: src/lectern-client/schema-rest-client.ts (hunk 2 above)
Comment (blabadi) [reply src/lectern-client/schema-rest-client.ts]: the name was never considered for stub schema not for any specific reason, like it fetches anything

File: src/lectern-client/schema-rest-client.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (blabadi) [reply src/lectern-client/schema-rest-client.ts]: updated .env.example to point you to the sample file

File: src/lectern-client/schema-rest-client.ts (hunk 2 above)
Comment (blabadi) [reply src/lectern-client/schema-rest-client.ts]: updated the file loader to consider the name

File: src/lectern-client/schema-rest-client.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (jaserud) [reply src/lectern-client/schema-rest-client.ts]: cool, I just got confused cause there is a stub-schema in `test/integration/clinical/stub-schema.json` following old stub-schema, but I guess that one doesn't really matter for clinical endpoints... :thinking:

**Pull request merged**